tests/testthat/test-textmodel_newsmap.R

```diff
@@ -34,10 +34,31 @@ test_that("test German dictionary and prediction work correctly", {
     )
 })
 
+
+test_that("test French dictionary and prediction work correctly", {
+    txt_fr <- c("Ceci est un article sur l'Irlande.")
+
+    toks_fr <- tokens(txt_fr)
+    toks_fr <- tokens_split(toks_fr, "'")
```

Comment:
@ClaudeGrasland please see https://github.com/quanteda/quanteda/issues/1610 and comment if possible.